d2-schemas/src/main/pegasus/com/linkedin/d2/D2RingProperties.pdl

```diff
@@ -0,0 +1,92 @@
+namespace com.linkedin.d2
+
+/**
+ * D2 hash ring properties used for load balancing.
+ */
+record D2RingProperties {
+
+  /**
+   * How fast should load balancer ramp up node that has been 100% degraded and is being recovered. Defaults to 1.0.
+   */
+  ringRampFactor: optional double
+
+  /**
+   * The number of points per weight given for a particular node in a hash ring. Defaults to 100.
+   */
+  pointsPerWeight: optional int
```

Comment:
Does this field still make sense for relative LB?

Reply:
Yes, see comment below.